lxqt-session/lxqt-config-session/lxqt-config-session_de.ts

```diff
@@ -272,7 +272,7 @@
     <message>
         <location filename="../../../userlocationspage.cpp" line="179"/>
         <source>LXQt Session Settings - User Directories</source>
-        <translation type="unfinished"></translation>
+        <translation>LXQt Sitzungseinstellungen - Nutzerverzeichnisse</translation>
```

Comment:
I'm aware that hyphens are frequently used in similar strings of the German translations but is this really accurate?
IMO strings like 'LXQt foobar' aren't a compound where the hyphen would indeed be mandatory but rather 'LXQt' is a kind of grammatical modifier to 'foobar' and hence a hyphen doesn't make sense. Also, wouldn't a hyphen in the English template be mandatory as well if those strings were meant to be a compound?